Control/public/configuration/Configuration.js

```diff
@@ -154,11 +154,7 @@ export default class Configuration extends Observable {
       // type number
       const valueNumber = parseInt(value);
       if (!isNaN(valueNumber)) {
-        if (field === 'onu-address' && valueNumber >= 0 && valueNumber <= (Math.pow(2, 31) - 1)) {
-          this.actionPanel.expertOptions[field] = valueNumber;
-        } else if (field === 'trigger-window-size' && value >= 0 && value <= 4095) {
-          this.actionPanel.expertOptions[field] = valueNumber;
-        } else if (field === 'cru-id' && valueNumber >= 0 && valueNumber <= (Math.pow(2, 31) - 1)) {
+        if (valueNumber >= 0 && valueNumber <= (Math.pow(2, 31) - 1)) {
```

Comment:
Change requested by the roc-commands team. Values will change from time to time and for the moment an upper limit is preferred